pyvocz/views.py

```diff
@@ -339,7 +338,7 @@ def event_qrcode(series_slug, date_slug):
         qr_byte_io = BytesIO()
         qr_img.save(qr_byte_io, 'PNG')
         qr_byte_io.seek(0)
-        cache.set(url, qr_byte_io, timeout=5*60)
+        cache.set(url, qr_byte_io, timeout=5 * 60)
```

Comment:
I couldn't find the rationale for this change in [PEP 8](https://www.python.org/dev/peps/pep-0008).

Reply:
Still it's visually readable.

Reply:
This is not that much about this particular case, but about a tool forcing something that's not actually in PEP8.